.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## Issue Overview
+<!-- This is a brief overview of the issue. --->
+
+## Expected Behavior
+<!-- If you're describing a bug, tell us what should happen -->
+<!-- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!-- If describing a bug, tell us what happens instead of the expected behavior -->
+<!-- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!-- Not obligatory, but suggest a fix/reason for the bug, -->
+<!-- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
```

Comment:
I'd reorder this, the sections should be in the order of:

`## Steps to Reproduce (for bugs)`
`## Expected Behavior`
`## Current Behavior`

It allows the issue reader to gain an overview of the events that led up to the "current behaviour" they see